### PR TITLE
make a duplicate checks script

### PR DESCRIPTION
**Pull request by freben**
Now checks all lockfiles

### CODE REVIEW

File: .github/workflows/verify_microsite.yml [hunk 1]
```diff
@@ -32,6 +32,10 @@ jobs:
         run: yarn install --immutable
         working-directory: microsite
 
+      - name: verify yarn dependency duplicates
+        run: yarn dedupe --check
+        run: node scripts/verify-lockfile-duplicates.js
```
Comment (Rugvip) [inline .github/workflows/verify_microsite.yml:37]: 👀

File: scripts/verify-lockfile-duplicates.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    if (stderr) {
+      console.error(stderr);
+    }
+
+    if (failed) {
+      if (!fix) {
+        const command = `yarn dedupe${
+          lockFile.directoryRelativeToProjectRoot === '.'
+            ? ''
+            : ` --cwd ${lockFile.directoryRelativeToProjectRoot}`
```
Comment (Rugvip) [inline scripts/verify-lockfile-duplicates.js:104]: 👌

File: package.json [hunk 3]
```diff
@@ -33,7 +33,8 @@
     "build-storybook": "yarn ./storybook run build-storybook",
     "techdocs-cli": "node scripts/techdocs-cli.js",
     "techdocs-cli:dev": "cross-env TECHDOCS_CLI_DEV_MODE=true node scripts/techdocs-cli.js",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "postinstall": "husky install"
```
Comment (freben) [inline package.json:37]: See https://typicode.github.io/husky/#/?id=yarn-2 - this is the way to do it in yarn 2 and 3

**Review by Rugvip — APPROVED** [review]
✨

File: .github/workflows/verify_microsite.yml (hunk 1 above)
Comment (freben) [reply .github/workflows/verify_microsite.yml]: 🦤

**Review by jhaals — APPROVED** [review]

**Pull request merged**